client/analytics/settings/historical-data/status.js

```diff
@@ -51,6 +50,4 @@ function HistoricalDataStatus( { importDate, status } ) {
 	);
 }
 
-export default useFilters( HISTORICAL_DATA_STATUS_FILTER )(
-	HistoricalDataStatus
-);
+export default HistoricalDataStatus;
```

Comment:
So this can no longer be filtered in any way, is that right?  Do we think it's unlikely to be filtered or is there another way to filter this?

Reply:
It can, the `statusLabels` are still filtered as they always have been earlier in the file:

https://github.com/woocommerce/woocommerce-admin/blob/b6d467d1b49251a9ee3f569ebccae3834dce32b9/client/analytics/settings/historical-data/status.js#L12